src/frontend/parser.mly

```diff
@@ -215,8 +215,8 @@ identifier:
   | TRUNCATE { build_id "T" $loc}
   | id_and_v = future_keyword
     {
-      let id, v = id_and_v in
-      Input_warnings.future_keyword id.name v $loc;
+      let id, _ = id_and_v in
+      (* Input_warnings.future_keyword id.name v $loc; *)
```

Comment:
You removed this to avoid a spurious warning on every use of the word `jacobian`. However,
1. If the commented out code is meant to be restored when `jacobian` becomes a true keyword then I'd like to see a `TODO(2.38)` marker.
2. The only future keyword is `jacobian` so reactivating this warning will not be necessary---it could just be removed now.
3. The future keyword warning would not have this "false positive" if it was on `delc_identifier` rather than `identifier`. And in my opinion that would make more sense anyway. The user does not need the warning on every use of the variable, warning about the declaration is sufficient. (Though note that a for-loop loop variable is another declaration site but does not go through `decl_identifier`)

Reply:
The future keyword mechanism was (re-)introduced specifically for jacobian in the last release, so I agree it is good to just remove now.